README.md

```diff
@@ -1 +1,22 @@
-# Paisley
\ No newline at end of file
+# Paisley
+
+The backend server needs to be started along with a celery worker used to handle background tasks within the query route. See below.
+
+Start/stop backend server:
+`sudo systemctl start test.service`
+`sudo systemctl stop test.service`
+
+Check backend server logs:
+`sudo systemctl status test.service`
```

Comment:
Recent logs are available using systemctl status test.service (or systemctl status celery.service)
FULL logs are available using `journalctl -u test.service`  (optional:  `--since "YYYY-MM-DD HH:MM:SS" --until "YYYY-MM-DD HH:MM:SS"`